ci/prow/make_config.go

```diff
@@ -53,41 +53,39 @@ const (
 
 // repositoryData contains basic data about each Knative repository.
 type repositoryData struct {
-	Name                string
-	EnableGoCoverage    bool
-	GoCoverageThreshold int
```

Comment:
You still need this, otherwise we can't change the threshold per job.

Reply:
Ditto.